container-guide/concepts/containers-podman.xml

```diff
@@ -403,14 +403,33 @@
 <screen>&prompt.user;podman run --rm -ti suse/sle15</screen>
 <note>
 <title>
-Solving network connection issues with rootful containers
+Fixing the lost network access to rootful containers
 </title>
 <para>
-Rootful containers may occasionally lose network connectivity due to another process deleting the firewall rules configured by &netavark;. Starting with version 1.9.0, &netavark; comes with the <systemitem>netavark-firewalld-reload.service</systemitem> service that solves the problem. Enable and start the service as follows:
+Rootful containers may occasionally become inaccessible from the outside. The
+issue is caused by firewalld reloading its permanent rules and discarding any
+temporary rules created by &podman;'s networking backend (either CNI or
+&netavark;).
+
+A temporary workaround is to reload the &podman; network using the
+<command>podman network reload --all</command> command. If you are using
```

Comment:
The current sentence flows from "A temporary workaround.." so I feel it's better to distinguish it with the permanent fix now offered via the service. Wdyt?
```suggestion
<command>podman network reload --all</command> command.

A permanent solution exists if you are using...
```